src/config.rs

```diff
@@ -231,6 +367,12 @@ static CONFIG_FILE_HELP: Lazy<String> = Lazy::new(|| {
     api_key = "<key-with-unlocks-scope>"
     lang = "<lang>"
 
+    [currencies]
+    ascended = <opportunity cost per item>
+    karma = <opportunity cost per karma>
+    um = <opportunity cost per um>
+    vm = <opportunity cost per vm>
```

Comment:
What do you think about using the full names here e.g. `unbound_magic`? It seems like there could be conflicts with other acronyms in the future and it'd be good to avoid having to make breaking changes to the config format in the future.

Reply:
Was trying to keep it terse, and figured future acronyms could adapt; both UM and VM are currently in common use, so those two at least should be fine. I was much more hesitant wrt RN, but wanted to keep it consistent. I originally had it as research_notes and didn't like it that way.